tools/run_presubmit

```diff
@@ -24,6 +24,10 @@ from typing import List, Callable, Optional, Tuple, Iterable
 
 REPO_ROOT = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# Check if we are inside a git hooks folder. If so go up one extra level.
```

Comment:
add a comment saying that this is also run as a git hook, so it's clearer
(not sure if you need it if a symlink)